bigquery_etl/query_scheduling/task.py

```diff
@@ -155,6 +155,7 @@ class Task:
     allow_field_addition_on_date: Optional[str] = attr.ib(None)
     destination_table: Optional[str] = attr.ib(default=DEFAULT_DESTINATION_TABLE_STR)
     is_python_script: bool = attr.ib(False)
+    execution_delay: Optional[str] = attr.ib(None)
```

Comment:
I added an `execution_delay` option here. For the hourly experiment enrollment aggregate task we want to make sure we account for potential BigQuery sink delays. The delay can be configured with this parameter in the `metadata.yaml` file.